src/main/kotlin/io/sc3/goodies/misc/DimmableLight.kt

```diff
@@ -0,0 +1,50 @@
+package io.sc3.goodies.nature
+
+import io.sc3.goodies.util.BaseBlock
+import net.minecraft.block.Block
+import net.minecraft.block.BlockState
+import net.minecraft.item.ItemPlacementContext
+import net.minecraft.state.StateManager
+import net.minecraft.state.property.IntProperty
+import net.minecraft.state.property.Properties
+import net.minecraft.util.math.BlockPos
+import net.minecraft.world.World
+import java.util.function.ToIntFunction
+
+fun createLightLevelFromPowerState(): ToIntFunction<BlockState> {
+  return ToIntFunction { state: BlockState -> state.get(Properties.POWER) ?: 0 }
+}
```

Comment:
Move this to be inside the companion object in `DimmableLight`, and use it like `DimmableLight.createLightLevel...` in `Registration`